config/schedule.yml

```diff
@@ -146,3 +146,7 @@ capture_query_stats:
   description: "Collects Postgres query stats for PGHero (runs every 5 minutes)"
   cron: "*/5 * * * *"
   class: "CaptureQueryStatsWorker"
+notify_about_published_articles:
```

Comment:
@maestromac I want to make sure that you're seeing the "runs every minute"; I hope that isn't too frequent.

Reply:
@maestromac We agreed on "every 5 minutes" with Jeremy :thinking: